src/api/goods.ts

```diff
@@ -5,15 +5,20 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => response.json())
+    .then(goods => goods);
 }
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      return goods
+        .sort((good1, good2) => good1.name.localeCompare(good2.name))
+        .slice(0, 5);
+    }); // sort and get the first 5
 };
 
 export const getRedGoods = () => {
   return getAll()
-    .then(goods => goods); // get only red
+    .then(goods => goods.filter(good => good.color === 'red')); // get only red
```

Comment:
you need just color here
```suggestion
    .then(goods => goods.filter(({ color }) => color === 'red'));
```